debian/rules

```diff
@@ -7,6 +7,7 @@ deb_version := $(shell dpkg-parsechangelog | sed -n 's/^Version: //p')
 
 override_dh_auto_configure:
 	mkdir -p build
+    ifndef CIRCLECI
```

Comment:
I wonder if it isn't better to do something like  (made up syntax):
```
ifndef CIRCLECI
RELEASE_FLAGS="-DNDEBUG -O2"
else
RELEASE_FLAGS="-O2"
cd build && cmake \
    -DCMAKE_CXX+FLAGS_RELEASE=${RELEASE_FLAGS} \
    ...
```
to avoid duplication and then forgetting to update one branch

Reply:
You're right. I've fixed this with the latest commit.

Reply:
@saarikoski-jules I think Yuri might have meant to avoid encoding the whole cmake command in both sides of the branch, since we might change that in the future and we'll 100% forget to keep the two branches in sync.

Reply:
yes I meant that. The latest commit is good

Reply:
Oops, of course. This is hopefully better.